deblur/parallel_deblur.py

```diff
@@ -0,0 +1,71 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015, The Deblur Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+from __future__ import division
+
+from os.path import basename, join, split, splitext
+from os import mkdir
+
+from bfillings.sortmerna_v2 import build_database_sortmerna
+from skbio.parse.sequences import parse_fasta
+
+from qiime.parallel.util import ParallelWrapper
+
+
+class ParallelDeblur(ParallelWrapper):
```

Comment:
So if we can use ```multiprocessing``` for launching jobs on clusters then probably we can avoid importing ```split_sequence_file_on_sample_ids_to_files``` and ```write_biom_table``` as well by writing our own methods?